tests/unit_test/llm_test.py

```
@@ -1097,25 +1492,30 @@
         }
 
         gpt3 = LLMProcessor(test_content.bot, DEFAULT_LLM)
-        mock_embedding.side_effect = [Exception("Connection reset by peer!"), litellm.EmbeddingResponse(**{'data': [{'embedding': embedding}]})]
+        text_embedding_3_small_embeddings = np.random.random(1536).tolist()
+        colbertv2_0_embeddings = [np.random.random(128).tolist()]
+        bm25_embeddings = {
+            "indices": [1850593538, 11711171],
+            "values": [1.66, 1.66]
+        }
 
+        embeddings = {
```

Ruff (0.8.2)

Local variable `embeddings` is assigned to but never used

Remove assignment to unused variable `embeddings`

(F841)